### ISSUE TITLE
Moving repo to an organization?

### ISSUE THREAD

**Issue by timschlechter** [bootstrap-tagsinput/bootstrap-tagsinput#189]
I'm getting loads of help lately with improving this plugin. @danielsvane , @kuraga and @rlidwka have helped me bringing this repo in better shape than I ever could have done ;-)

Since I don't want to get all the credits with this repo hosted on my account, I suggest moving it to an github organization. This also ensures better maintenance, since there are times I simply can't find the time to invest in it.

I just made https://github.com/bootstrap-tagsinput, and intend to move it over there and make @danielsvane , @kuraga and @rlidwka member.

Is this a good idea, or do you guys see any problems in this plan?

**Comment by jcrben** [comment]
https://github.com/corejavascript/typeahead.js is an organization for a fork of the related (and abandoned) typeaheadjs library and it seems like it could be good for these two libraries to live under the same organization.

I don't know how the CoreJavascript group will shake out - I got added as a Collaborator but may not have time to chip in. But it does make sense to have organization which has some members and a brand (similar to Apache).
